### ISSUE TITLE
ValueError in label.remove('Your Groups')

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#73]
```
> File 
"/usr/lib/python2.6/dist-packages/gphotoframe/plugins/flickr/__init__.py", line 
182, in _label
>
> label.remove('Your Groups')
>
> ValueError: list.remove(x): x not in list

```

Original issue reported on code.google.com by `yendo0206` on 24 Oct 2010 at 9:27

**Comment by GoogleCodeExporter** [comment]
```
If you log in the flickr service from the plugins tab, this bug does not occur.

```

Original comment by `yendo0206` on 24 Oct 2010 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision a37a3f3e0d.

```

Original comment by `yendo0206` on 24 Oct 2010 at 9:50
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thank you very much! Comment 1 resolved the problem with gphotoframe1.1 and 
also the icons are back!
```

Original comment by `cri.pe...@gmail.com` on 24 Oct 2010 at 11:57
